test/puppetlabs/puppetdb/fixtures.clj

```diff
@@ -38,18 +37,6 @@
       (migrate!)
       (f))))
 
-(defn without-db-var
-  "Binds the java.jdbc dtabase connection to nil. When running a unit
-   test using `with-test-db`, jint/*db* will be bound. If the routes
-   being tested don't explicitly bind the db connection, it will use
-   one bound in with-test-db. This causes a problem at runtime that
-   won't show up in the unit tests. This fixture can be used around
-   route testing code to ensure that the route has it's own db
-   connection."
-  [f]
-  (binding [jint/*db* nil]
```

Comment:
Does the deprecated stuff not use this anymore? I'm confused as to why this would need to be removed as part of this patch.